openedx/core/lib/grade_utils.py

```diff
@@ -42,3 +43,20 @@ def is_score_higher_or_equal(earned1, possible1, earned2, possible2, treat_undef
     """
     is_higher_or_equal, _, _ = compare_scores(earned1, possible1, earned2, possible2, treat_undefined_as_zero)
     return is_higher_or_equal
+
+
+def round_away_from_zero(number, digits=0):
+    """
+    Round numbers using the 'away from zero' strategy as opposed to the
+    'Banker's rounding strategy.'  In python 2 this is how round used to
+    work but python 3 uses Banker's roundig as the implementation of round.
```

Comment:
"roundig", but also, you might want to explicitly explain the naming. It's only about what happens with .5, not that everything is rounded away from zero.  Half the numbers move closer to zero.